src/lib/util/Security/ModerationActions.ts

```diff
@@ -447,6 +471,36 @@ export class ModerationActions {
 		return this.removeStickyRole(moderatorID, id, mutedRole);
 	}
 
+	private async nicknameUser(rawOptions: ModerationActionOptions, nickname: string) {
+		try {
+			const member = await this.guild.members.fetch(rawOptions.user_id);
+			return await member.setNickname(nickname);
+		} catch (error) {
+			if (error.code === APIErrors.UnknownMember) throw this.guild.language.tget('ACTION_REQUIRED_MEMBER');
+			throw error;
+		}
```

Comment:
There's an utility that takes a Promise and makes it throw on unmatching APIErrors codes, returning `null` on those that match the codes passed, you can use it instead of wrapping this on a try/catch 👍

Reply:
This one, specifically:
https://github.com/skyra-project/skyra/blob/0a4ca2ddd9669b41cb5f67a387d59a076f49da32/src/lib/util/util.ts#L524-L531